autobump-maven/src/main/java/com/github/autobump/maven/model/MavenDependencyResolver.java

```diff
@@ -22,6 +33,36 @@ public MavenDependencyResolver() {
         Model model = mavenModelAnalyser.getModel(workspace);
         Set<Dependency> dependencies = getDependencies(model);
         dependencies.addAll(getPlugins(model));
+        var modules = model.getModules();
+        if (!modules.isEmpty()) {
+            try {
+                dependencies.addAll(resolveModules(workspace));
+            } catch (IOException e) {
+                throw new UncheckedIOException(e);
+            }
+        }
+        return dependencies;
+    }
+
+    public Set<Dependency> resolveModules(Workspace workspace) throws IOException {
+        var dependencies = new HashSet<Dependency>();
+        Files.walkFileTree(Path.of(workspace.getProjectRoot()),
+                Set.of(),
+                2,
+                new SimpleFileVisitor<Path>(){
+                @Override
+            public FileVisitResult visitFile(Path file, BasicFileAttributes attrs) {
+                    if (!file.toString().equals(workspace.getProjectRoot() + File.separator + FILENAME) &&
+                             file.getFileName().toString().equals(FILENAME)){
+                        Workspace ws = new Workspace(file
```

Comment:
Workspace is a wrapper around a (temporary) directory and provides access to files within that directory. Maybe the Workspace can encapsulate logic like this to:
- visit files within the directory
- read files within the directory by giving a "sub-path"

This will also keep the MavenDependencyResolver a bit smaller and focussed on resolving dependencies instead of messing with files.
At last, it will probably make it easier to test MavenDependencyResolver by mocking the Workspace. What if a file is missing? What if reading a file fails with an IOException, ...